net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml

```diff
@@ -0,0 +1,44 @@
+<form>
+    <field>
+        <id>user.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>This will enable or disable the user account. Allowed characters are 0-9, a-z, A-Z, and ._-</help>
+    </field>
+    <field>
+        <id>user.username</id>
+        <label>Username</label>
+        <type>text</type>
+        <help>Set the unique username for the user. Allowed characters are 0-9, a-z, A-Z, and ,._-!$%/()+#= with up to 128 signs</help>
+    </field>
+    <field>
+        <id>user.password</id>
+        <label>Password</label>
+        <type>text</type>
+        <help>Set the password for the user.</help>
+    </field>
+    <field>
+        <id>user.description</id>
+        <label>Description</label>
+        <type>text</type>
+        <help>Surname, Lastname, or anything you need to describe.</help>
```

Comment:
Surname, and "last name" are the same. Vorname is "given name" if you meant that. No capitalisation of works in English beyond the first word mostly.